hw3/02-charts.js

```diff
@@ -0,0 +1,114 @@
+const backgroundColors = [
+  "rgba(54, 162, 235, 0.8)",
+  "rgba(255, 206, 86, 0.8)",
+  "rgba(255, 99, 132, 0.8)",
+  "rgba(75, 192, 192, 0.8)",
+  "rgba(153, 102, 255, 0.8)",
+  "rgba(255, 159, 64, 0.8)",
+  "rgba(199, 199, 199, 0.8)",
+  "rgba(83, 102, 255, 0.8)",
+  "rgba(40, 159, 64, 0.8)",
+  "rgba(210, 199, 199, 0.8)",
+  "rgba(78, 52, 199, 0.8)",
+];
+
+const borderColors = [
+  "rgba(54, 162, 235, 1)",
+  "rgba(255, 206, 86, 1)",
+  "rgba(255, 99, 132, 1)",
+  "rgba(75, 192, 192, 1)",
+  "rgba(153, 102, 255, 1)",
+  "rgba(255, 159, 64, 1)",
+  "rgba(159, 159, 159, 1)",
+  "rgba(83, 102, 255, 1)",
+  "rgba(40, 159, 64, 1)",
+  "rgba(210, 199, 199, 1)",
+  "rgba(78, 52, 199, 1)",
+];
+
+// url for the Thrones API
+const url = "https://thronesapi.com/api/v2/Characters";
+
+async function renderChart() {
+  console.log("in render chart");
+  const donutChart = document.querySelector(".donut-chart");
+
+  let houseCount = {
+    Stark: 0,
+    Arryn: 0,
+    Baratheon: 0,
+    Tully: 0,
+    Greyjoy: 0,
+    Lannister: 0,
+    Tyrell: 0,
+    Martell: 0,
+    Targaryen: 0,
+  };
+  const characters = await  getCharacters();
+  console.log(characters);
+
+  characters.forEach((character) => {
```

Comment:
A way to avoid chained else if statements like this would be to create a map that can use the family name as a key and the amount of people in that family as the value. This would also capture all people the API has.